rules/busybox.bzl

```diff
@@ -641,14 +645,33 @@ def _compile(
     )
     args.add("--output", out_file)
 
+    optional_outputs = []
+    if out_class_jar:
+        args.add("--classJarOutput", out_class_jar)
+        optional_outputs.append(out_class_jar)
+    if out_r_txt:
+        args.add("--rTxtOut", out_r_txt)
+        optional_outputs.append(out_r_txt)
+
+    args.add("--targetLabel", ctx.label)
+    optional_inputs = []
+    if manifest:
+        args.add("--manifest", manifest)
+        optional_inputs = [manifest]
+    if android_jar:
+        args.add("--androidJar", android_jar)
+        optional_inputs.append(android_jar) 
+
+    _disable_warnings(args)
```

Comment:
`disable_warnings()` doesn't seem to exist on my end. Where is this function from?